rail_scripts/prepare-pz-test

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
```

Comment:
I would leave this script inside the htcondor directory. It is specific for runs with Condor and has dependencies on files that are in `scheduler_examples/htcondor/`.

Reply:
I agree. Also, update setup instructions so it is included in PATH (the reason I put it inside rail_scripts originally).